test/units/modules/network/nxos/test_nxos_ospf_vrf.py

```diff
@@ -54,16 +54,75 @@ def test_nxos_ospf_vrf_present(self):
                              timer_throttle_lsa_start=60,
                              timer_throttle_lsa_hold=1100,
                              timer_throttle_lsa_max=3000,
+                             bfd='enable',
                              state='present'))
         result = self.execute_module(changed=True)
         self.assertEqual(sorted(result['commands']),
                          sorted(['router ospf 1',
                                  'vrf test',
                                  'timers throttle lsa 60 1100 3000',
                                  'timers throttle spf 50 1000 2000',
-                                 'vrf test']))
+                                 'bfd',
+                                 ]))
 
     def test_nxos_ospf_vrf_absent(self):
         set_module_args(dict(ospf=1, vrf='test', state='absent'))
         result = self.execute_module(changed=False)
         self.assertEqual(result['commands'], [])
+
+    def test_bfd_1(self):
+        self.get_config.return_value = 'router ospf 1\n  bfd\nrouter ospf 2'
```

Comment:
If you don't want to put the router data in a separate file, I would prefer the following for readability

```python
return_value = """
router ospf 1
  bfd
router ospf 2
"""

I know the alignment looks a bit odd but it's easier to read IMO.

Reply:
Can't. The config needs to be in a format that `CustomNetworkConfig` can handle (which expects a certain indentation) - otherwise it breaks the call to `netcfg.get_section()`.

Reply:
*sigh* Oh well...